### PR TITLE
Test scripts for market manager

### PR DESCRIPTION
**Pull request by Ravleen-Solulab**
This PR contains test scripts for market_manager_abci

### CODE REVIEW

File: LICENSE
Comment (Adamantios) [inline LICENSE:1]: ?

File: gitleaks
Comment (Adamantios) [inline gitleaks:1]: Please remove.

File: gitleaks_8.18.4_linux_x64.tar.gz
Comment (Adamantios) [inline gitleaks_8.18.4_linux_x64.tar.gz:1]: Please remove.

File: packages/__init__.pyc
Comment (Adamantios) [inline packages/__init__.pyc:1]: Please remove.

File: packages/valory/skills/__init__.pyc
Comment (Adamantios) [inline packages/valory/skills/__init__.pyc:1]: Please remove.

File: packages/valory/__init__.pyc
Comment (Adamantios) [inline packages/valory/__init__.pyc:1]: Please remove.

File: packages/valory/skills/check_stop_trading_abci/__init__.pyc
Comment (Adamantios) [inline packages/valory/skills/check_stop_trading_abci/__init__.pyc:1]: Please remove.

File: packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    def test_run(self, test_case: RoundTestCase) -> None:
+        """Run tests."""
+        self.run_test(test_case)
+
+    def run_test(self, test_case: RoundTestCase) -> None:
+        """Run tests."""
+        if test_case.event == Event.SKIP_TRADING:
+            self.test_positive_votes()
+        elif test_case.event == Event.NO_MAJORITY:
+            self.test_negative_votes()
```
Comment (Adamantios) [inline packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py:237]:
```suggestion
        if test_case.event == Event.SKIP_TRADING:
            self.test_positive_votes()
        elif test_case.event == Event.NO_MAJORITY:
            self.test_negative_votes()
```

File: packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                round_payloads=get_participant_to_votes(self.participants, vote=False),
+                synchronized_data_update_fn=lambda _synchronized_data, _: _synchronized_data.update(
+                    participant_to_votes=get_participant_to_votes_serialized(
+                        self.participants, vote=False
+                    )
+                ),
+                synchronized_data_attr_checks=[],
+                exit_event=self._event_class.DONE,
+            )
+        )
```
Comment (Adamantios) [inline packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py:182]: Please eliminate code dup.

**Review by Adamantios — CHANGES_REQUESTED** [review]
All these formatting changes make it hard to review the code. Please revert them.
Also, please resolve the conflicts with `main`.

**Comment by Ravleen-Solulab** [comment]
I have reverted the changes.

**Pull request closed without merging**